definitions/mw.lua

```diff
@@ -1018,9 +1018,22 @@ function mw.ext.TeamLiquidIntegration.resolve_redirect(name) return name end
 
 mw.ext.TeamTemplate = {}
 
+---@class teamTemplateData
+---@field templatename string
+---@field historicaltemplate string?
+---@field shortname string
+---@field name string
+---@field bracketname string
+---@field page string
+---@field icon string
+---@field image string
+---@field imagedark string
+---@field legacyimage string
+---@field legacyimagedark string
```

Comment:
Multiline annotations should be support since latest version of LuaLS, so the class-workaround should no longer be needed

Reply:
using it in sep module though, hence added it as a class
if you prefer multi line lmk

Reply:
Can use an alias instead of a class